docs/v0.5.x/aliasing-plugins.md

```diff
@@ -1,3 +1,18 @@
 ---
 title: Aliasing Plugins
 ---
+
+To include a plugin twice, alias it using a colon.
+
+```javascript
+plugins: ["s3-assets:foo-assets",
+          "s3-assets:bar-assets",
+          "s3-index",
+          "notify-slack"]
+```
+
+The name specified after the colon will be passed as the `name` property
+of the `options` argument to the addon's `createDeployPlugin` method. Plugins
+should use their name to retrieve configuration values. In this example,
+the foo-assets instance of the s3-assets plugin could have different configuration
+than the bar-assets instance does.
```

Comment:
This page should include a configuration file snippet showing where `plugins` goes and how the configuration would look.